src/popup/components/InputSelectAsset.vue

```diff
@@ -25,31 +28,37 @@ export default {
     focused: Boolean,
     showTokensWithBalance: Boolean,
   },
-  computed: {
-    displayToken() {
-      if (!this.value) return '';
-      const { symbol } = this.value;
+  setup(props, { emit }) {
+    const { openModal } = useModals();
+
+    const displayToken = computed(() => {
+      if (!props.value) return '';
+      const { symbol } = props.value;
       return `${String(symbol).substring(0, 11)}${symbol.length > 11 ? '...' : ''}`;
-    },
-  },
-  methods: {
-    handleChange(token) {
-      this.$emit('input', token);
-    },
-    openAssetSelector() {
-      if (this.disabled) return;
+    });
+
+    function handleChange(token: IToken) {
+      emit('input', token);
+    }
+
+    function openAssetSelector() {
+      if (props.disabled) return;
+
+      openModal(MODAL_ASSET_SELECTOR, {
+        selectedToken: props.value,
+        showTokensWithBalance: props.showTokensWithBalance,
+        resolve: (token) => token,
+      })
+        .then((token) => handleChange(token));
```

Comment:
remove additional space
```suggestion
     .then((token) => handleChange(token));
```

Reply:
Same comment as the one here: https://github.com/aeternity/superhero-wallet/pull/1948#discussion_r1147658838